src/local_scheduler/local_scheduler.cc

```diff
@@ -712,10 +742,21 @@ void reconstruct_put_task_update_callback(Task *task,
         /* (2) The current local scheduler for the task is dead. The task is
          * lost, but the task table hasn't received the update yet. Retry the
          * test-and-set. */
+#if !RAY_USE_NEW_GCS
         task_table_test_and_update(state->db, Task_task_id(task),
                                    current_local_scheduler_id, Task_state(task),
                                    TASK_STATUS_RECONSTRUCTING, NULL,
                                    reconstruct_put_task_update_callback, state);
+#else
+        RAY_CHECK_OK(gcs::TaskTableTestAndUpdate(
+            &state->gcs_client, Task_task_id(task), current_local_scheduler_id,
+            Task_state(task), SchedulingState_RECONSTRUCTING,
+            [task, user_context](gcs::AsyncGcsClient *, const ray::TaskID &,
```

Comment:
This kind of thing might be more readable if you make it a variable with a name.

Same comment in other places as well.

Reply:
Zongheng asked me to remove the variable names, let's leave it that way now

Reply:
If you add names, please do prefix them with "unused_".

See examples:
+ https://github.com/tensorflow/tensorflow/blob/579125e87af201ae6b6fa872b6dc3f3ecb400de9/tensorflow/core/ops/data_flow_ops.cc#L166
+ https://github.com/tensorflow/tensorflow/blob/754048a0453a04a761e112ae5d99c149eb9910dd/tensorflow/python/platform/status_bar.py#L23